pep-0458.txt

```diff
@@ -828,44 +828,26 @@ private cryptographic keys (belonging to any of the PyPI roles) are
 compromised.  The leftmost column lists the roles (or a combination of roles)
 that have been compromised, and the columns to its right show whether the
 compromised roles leave clients susceptible to malicious updates, a freeze
-attack, or metadata inconsistency attacks.
+attack, or metadata inconsistency attacks. Note that if the timestamp, snapshot,
+and bin-n roles are stored in the same online location, a compromise of one
+means they will all be compromised. Therefore the table considers these
+roles together. A version of this table that considers these roles separately
+is included in PEP 480 [26]_.
 
 +-----------------+-------------------+----------------+--------------------------------+
 | Role Compromise | Malicious Updates | Freeze Attack  | Metadata Inconsistency Attacks |
 +=================+===================+================+================================+
-| timestamp       | NO                | YES            | NO                             |
-|                 | snapshot and      | limited by     | snapshot needs to cooperate    |
-|                 | targets or any    | earliest root, |                                |
-|                 | of the bins need  | targets, or    |                                |
-|                 | to cooperate      | bin expiry     |                                |
-|                 |                   | time           |                                |
+| targets         | NO                                                                  |
+| **OR**          | timestamp and snapshot need to cooperate                            |
+| bins            |                                                                     |
 +-----------------+-------------------+----------------+--------------------------------+
-| snapshot        | NO                | NO             | NO                             |
-|                 | timestamp and     | timestamp      | timestamp needs to cooperate   |
-|                 | targets or any of | needs to       |                                |
-|                 | the bins need to  | cooperate      |                                |
-|                 | cooperate         |                |                                |
+| timestamp       | YES                                                                 |
+| **AND**         | limited by earliest root, targets, or bins metadata expiry time     |
+| snapshot        |                                                                     |
+| **AND**         |                                                                     |
+| bin-n           |                                                                     |
```

Comment:
Should we stress that adding targets **OR*** bins has the same outcome somehow?

Reply:
I could add another row for this, but it would need to have parenthesis to be clear which I think is more likely to confuse the reader, ex: (targets OR bins) AND timestamp AND snapshot AND bin-n